src/calculateRentalCost.js

```diff
@@ -3,8 +3,21 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
+  const discount3 = 20;
+  const discount7 = 50;
+  const totalAmount = days * dayCost;
+
+  if (days < 3) {
+    return totalAmount;
+  } else if (days < 7) {
```

Comment:
Avoid using 'else if' after a 'return' statement. You can simply use an 'if' statement here, as the function will exit if the first 'if' condition is met.